lib/index.js

```diff
@@ -40,6 +40,18 @@ const endMeasure = (baseName) => {
   performance.clearMarks(`${baseName}-start`);
 };
 
+const ccBinaryFound = (execPath, err) => {
+  if (err.code === 'ENOENT' && !badPaths.has(execPath)) {
+    const msg = 'codeclimate binary not found! Installation ' +
+      'instructions at http://github.com/codeclimate/codeclimate';
+    atom.notifications.addError(msg);
+    // Only notify once per path
+    badPaths.add(execPath);
+    return false;
+  }
+  return true;
```

Comment:
For example if the user mis-configures things and puts a directory in the `executablePath` setting, you get `EDIR` or `EACCESS` errors.